src/util/testtypes.jl

```diff
@@ -28,7 +28,7 @@ AccessCountDiskArray(a; chunksize=size(a),batchstrategy=DiskArrays.ChunkRead(Dis
 Base.size(a::AccessCountDiskArray) = size(a.parent)
 
 # Apply the all in one macro rather than inheriting
-DiskArrays.@implement_diskarray AccessCountDiskArray
+DiskArrays.@implement_diskarray_skip_zip AccessCountDiskArray
```

Comment:
```suggestion
```

This isn't needed, it's a subtype already

Reply:
Ah, ok, thanks I'll update that